plugins/filter/accumulate.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Max Gautier <mg@max.gautier.name>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = '''
+  name: accumulate
+  short_description: Produce a list of accumulated sums of the input list contents.
+  version_added: 10.0.0
```

Comment:
10.0.0 has already been released, the next release with new features will be 10.1.0:
```suggestion
  version_added: 10.1.0
```